### PR TITLE
DDF-3350 relocated registry content to configuring section of docs

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Moved configuring steps for Registry app to be a sub-section of Configuring Federation.

#### Who is reviewing it? 
@mcalcote @vinamartin @ahoffer @garrettfreibott 

#### Choose 2 committers to review/merge the PR. 
@clockard
@lessarderic
@pklinef
@ricklarsen - Documentation
@rzwiefel
@shaundmorris

#### How should this be tested? (List steps with links to updated documentation)

Review content section. No changes made to templates or process.

#### What are the relevant tickets?
[DDF-3350](https://codice.atlassian.net/browse/DDF-3350)

#### Screenshots (if appropriate)

[documentation.pdf](https://github.com/codice/ddf/files/1569692/documentation.pdf)

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/docs/src/main/resources/content/_configuring/configuring-identity-node.adoc [hunk 1]
```diff
@@ -0,0 +1,286 @@
+:title: Configuring Identity Node
+:type: subConfiguration
+:status: published
+:parent: Federating through a Registry
+:order: 00
+:summary: Connfiguring identity node.
```
Comment (mcalcote) [inline distribution/docs/src/main/resources/content/_configuring/configuring-identity-node.adoc]: ❗️ Configuring

File: distribution/docs/src/main/resources/content/_configuring/configuring-identity-node.adoc [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+|String
+|
+
+|Binding Version
+|This binding version
+|
+|
+
+|Access URL
+|The url used to access this binding
```
Comment (mcalcote) [inline distribution/docs/src/main/resources/content/_configuring/configuring-identity-node.adoc]: ✏️ URL

File: distribution/docs/src/main/resources/content/_configuring/publishing-to-node.adoc [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+:order: 01
+:summary: Publishing to other nodes.
+
+== {title}
+
+Send details about the local ${branding} instance to other nodes.
+
+. Navigate to the *Remote Registries* tab in either ${ddf-registry} or ${ddf-catalog} application.
+. Click *Add* to add a remote registry.
+. Enter Registry Service (CSW) Url.
```
Comment (mcalcote) [inline distribution/docs/src/main/resources/content/_configuring/publishing-to-node.adoc]: ✏️ URL

File: distribution/docs/src/main/resources/content/_configuring/configuring-identity-node.adoc [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+|This organization's name
+|string
+|yes
+
+|Address
+|This organization's primary address
+|Expand to enter address information
+|yes
+
+|TelephoneNumber
```
Comment (mcalcote) [inline distribution/docs/src/main/resources/content/_configuring/configuring-identity-node.adoc]: ❗️ Telephone Number

File: distribution/docs/src/main/resources/content/_configuring/configuring-identity-node.adoc (hunk 1 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_configuring/configuring-identity-node.adoc]: fixed

File: distribution/docs/src/main/resources/content/_configuring/configuring-identity-node.adoc (hunk 2 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_configuring/configuring-identity-node.adoc]: fixed.

File: distribution/docs/src/main/resources/content/_configuring/configuring-identity-node.adoc (hunk 4 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_configuring/configuring-identity-node.adoc]: fixed.

File: distribution/docs/src/main/resources/content/_configuring/publishing-to-node.adoc (hunk 3 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_configuring/publishing-to-node.adoc]: FIXED

File: distribution/docs/src/main/resources/content/_configuring/configuring-identity-node.adoc [hunk 5]
```diff
@@ -0,0 +1,286 @@
...
+|
+|
+
+|Service Binding Type
+|The binding type for the service
+|
+|
+
+|URL Property Key
+|Property that the accessURI value should be put into for source creation
```
Comment (shaundmorris) [inline distribution/docs/src/main/resources/content/_configuring/configuring-identity-node.adoc]: ❓ Should this be "accessURL"?

**Comment by ricklarsen** [comment]
I made a few changes per previous content and have not updated the attached pdf

File: distribution/docs/src/main/resources/content/_configuring/configuring-identity-node.adoc (hunk 5 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_configuring/configuring-identity-node.adoc]: fixed

**Comment by ricklarsen** [comment]
build now

**Pull request merged**